### ISSUE TITLE
FA > 5.7.0 incompatible with Sketch and Illustrator files created with 5.6.1

### ISSUE THREAD

**Issue by jasonheppler** [FortAwesome/Font-Awesome#14994]
### The issue

Upgraded from FA Pro 5.6.1 to FA Pro 5.8.1 and the new font is not recognized in my Sketch files. I followed the procedure recommended by support for removing and adding FA in Mac OS several times with the same results.

Based on the change logs I believe the font name change noted in the 5.7.0  is resulting in Sketch seeing the renamed fonts in 5.8.1 as new fonts. 

### Expectations

I expected the upgrade to be seamless as it has been in the past. I'm reporting the issue because this change is going to consume a lot of time for our team even with a font replacement plugin for sketch and I would like to make the FA team aware of the issue so they can let the community know if there is a work around and/or so the FA team can try to avoid a name change in the future if it is the cause of this compatibility issue.

### What version and implementation are you using?

Version: 5.8.1 (I assume all versions from 5.7.0 will be incompatible with previous versions).
On the Desktop in Sketch and Illustrator on a Mac

### Reproducible test case

I can remove/install (switch between) 5.6.1 and 5.8.1 and in Sketch the text elements created with one or the other isn't compatible with the other but is compatible with the font used to create the text.

Video demonstrating Issue in Sketch file after installing 5.8.1 (updated two of 1,000 text fields):
http://heppler.com/FontAwesome/01%20FA%205.8.1%20not%20compatable%20with%20Sketch%20files%20created%20with%20FA%205.6.1.mp4

Video demonstrating revert to 5.6.1 and checking the two updated fields to confirm they are not compatible with the older 5.6.1:
http://heppler.com/FontAwesome/02%20Reverted%20to%205.6.1%20and%20checked%20Sketch%20file%20to%20show%20reverse%20problem%20applies.mp4

I am happy to provide additional information as needed.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for this detailed report.

> I expected the upgrade to be seamless as it has been in the past.

Sorry for the breaking change. I understand that this does not follow semver and while we are trying our best to avoid breaking change, we had to do fix a major issue for app developers (#13900). The breaking change is documented in the upgrading guide available at: https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md#56x-to-570

>  I would like to make the FA team aware of the issue so they can let the community know if there is a work around and/or so the FA team can try to avoid a name change in the future if it is the cause of this compatibility issue.

That's fine. I'm leaving this open, waiting for feedback from the community. Maybe someone could help

@sensibleworld do you know any technique to speed up this kind of upgrade?

**Comment by jasonheppler** [comment]
Appreciate the reply and assistance. Confirmation of the issue helps immensely, glad to know I'm not going crazy. I wasn't able to find others talking about this issue and the Font Awesome chat support provided steps for removing/reinstalling the font when I reported the issue which only added to the confusion. 

Just my personal opinion but I would revise the 5.7.0 notes to more clearly communicate the font name change applies to all of the font weights and to direct users to a support article explaining the issue, why the change was made, and explaining that while it may be painful it hopefully won't happen again in the future. For example:

"The PostScript names have been changed to add a hypen between the font name and the weight (for example: FontAwesome5ProSolid to FontAwesome5Pro-Solid). This was an important change to ensure compatibility with tooling such as XCode. The good news is that we don't anticipate any further name changes. (Support Article link)"

I'm excited to change the built-in "Missing Fonts" feature noted above and I will report back. Thank you for the support and suggestions!
